hummingbot/templates/conf_global_TEMPLATE.yml

```diff
@@ -170,6 +170,7 @@ send_error_logs: null
 instance_id: null
 log_level: INFO
 debug_console: false
+debug_console_host: localhost
```

Comment:
Since the template file is changed, the version in line 6 has to be updated too

Reply:
Thanks for the feedback, now fixed!